### ISSUE TITLE
windows x64 GPU build broken - 2019-02-12

### ISSUE THREAD

**Issue by gatapia** [dmlc/xgboost#4125]
Current head throws the following error on build:
```
c:\xgboost\src\common\./device_helpers.cuh(1052): error C3016: 'shard': index variable in OpenMP 'for' statement must have signed integral type [c:\dev\libs\xgboost\build_64_gpu\gpuxgboost.vcxproj]
```

Full build log with errors attached, this has multiple builds, some of which succeed but just search for FAILED.
[nightly_build_log.txt](https://github.com/dmlc/xgboost/files/2853215/nightly_build_log.txt)

**Comment by RAMitchell** [comment]
Thanks, let me fix this today. Hopefully we get CI up shortly.

**Comment by trivialfis** [comment]
Addressed in #4139.
